packages/css/src/types.ts

```diff
@@ -602,7 +602,7 @@ export interface ThemeStyles {
   [key: string]: ThemeUIStyleObject | undefined
 }
 
-export function getUseRootStyles(theme: Theme = {}): any {
+export function getUseRootStyles(theme: Theme = {}) {
```

Comment:
Actually... Why is this in types.ts file?

Reply:
I was expecting this one. I didn't find a better place where to put this, also not being familiar with the project.
Where would you put it?

Reply:
This is the only piece of executable runtime code in `types.ts`. I'd probably stick it in `index.ts`. It's growing, but I don't think we have to split it into multiple files yet.